sale_order_invoice_amount/models/sale_order.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import api, fields, models
+from odoo.tools.misc import formatLang
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    invoiced_amount = fields.Monetary(
+        compute="_compute_invoice_amount",
+        store=True,
```

Comment:
I think it would be nice to give it value inside an install hook in case someone decide to install this addon with a lot of sale orders.